vaporetto.pyi

```diff
@@ -0,0 +1,24 @@
+from collections.abc import Iterable
+from collections.abc import Iterator
+
+class Token:
+    def end(self) -> int: ...
+    def n_tags(self) -> int: ...
+    def start(self) -> int: ...
+    def surface(self) -> str: ...
+    def tag(self, index: int) -> str | None: ...
```

Comment:
```suggestion
    def tag(self, index: int) -> Optional[str]: ...
```
`X | None` is an alternative syntax for `Union[X, None]`, and [the official document](https://mypy.readthedocs.io/en/stable/kinds_of_types.html#optional-types-and-the-none-type) says 

> (Optional[X] is the preferred shorthand for Union[X, None])

Reply:
@kampersanda  `Optional[T]` is better than `Union[T, None]`, but the document does not say that `Optional[T]` is better than `T | None`.

In fact, `| None` is used in the typeshed project.
https://github.com/python/typeshed/blob/main/stubs/D3DShot/d3dshot/d3dshot.pyi

Reply:
@vbkaisetsu I see.